src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -318,14 +318,18 @@ public IActionResult TotalEmployees()
         [Authorize(Policy = PolicyNames.CanEditReport)]
         public IActionResult PostTotalEmployees(bool? hasMinimumEmployeeHeadcount)
         {
-            if (!hasMinimumEmployeeHeadcount.HasValue)
-                return new RedirectResult(Url.Action("TotalEmployeesConfirmationRequired", "Report"));
-
             if (hasMinimumEmployeeHeadcount == false)
                 return new RedirectResult(Url.Action("ReportNotRequired"));
 
             var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
 
+            if (!hasMinimumEmployeeHeadcount.HasValue)
```

Comment:
have we agreed with design to validate here as it is technically inconsistent with the rest of the question sets. The consistent approach would be to simply check no value and redirect back to the question list I think